docs/offline-mode.md

```diff
@@ -59,17 +59,17 @@ The offline database flag `--experimental-offline` causes OSV-Scanner to scan yo
 osv-scanner --experimental-offline ./path/to/your/dir
 ```
 
-## Local database option
+## Download offline databases option
 
-The local database flag `--experimental-local-db` causes OSV-Scanner to download or update your local database and then scan your project against it.
+The download offline databases flag `--experimental-download-offline-databases` causes OSV-Scanner to download or update your local database and then scan your project against it. This option only works when you also set the offline flag.
```

Comment:
nit: I would phrase this more as "allowed" rather than "causes", as I think that aligns more with the theme - we're trying to be as offline as possible, but sometimes we can't, this flag _allows_ the scanner to be a little less offline in those cases

you could frame it something like

```suggestion
The download offline databases flag `--experimental-download-offline-databases` allows OSV-Scanner to download or update your local database when running in offline mode, to make it easier to get started. This option only works when you also set the offline flag.
```